### PR TITLE
Docs on default ports.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

**Comment by fjy** [comment]
The announcer test is terribad:
Tests run: 5, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.719 sec <<< FAILURE! - in io.druid.curator.announcement.AnnouncerTest
testSessionKilled(io.druid.curator.announcement.AnnouncerTest)  Time elapsed: 3.93 sec  <<< ERROR!
org.apache.zookeeper.KeeperException$NoNodeException: KeeperErrorCode = NoNode for /test1
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:111)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:51)
	at org.apache.zookeeper.ZooKeeper.getData(ZooKeeper.java:1155)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:302)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$4.call(GetDataBuilderImpl.java:291)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.GetDataBuilderImpl.pathInForeground(GetDataBuilderImpl.java:287)
	at org.apache.curator.framework.imps.GetDataBuilderImpl.forPath(GetDataBuilderImpl.java:279)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$1.forPath(GetDataBuilderImpl.java:104)
	at org.apache.curator.framework.imps.GetDataBuilderImpl$1.forPath(GetDataBuilderImpl.java:64)
	at io.druid.curator.announcement.AnnouncerTest.testSessionKilled(AnnouncerTest.java:170)

File: docs/content/tutorials/cluster.md [hunk 1]
```diff
@@ -258,6 +258,25 @@ Keep -XX:MaxDirectMemory >= numThreads*sizeBytes, otherwise Druid will fail to s
...
+If you're using a firewall or some other system that only allows traffic on specific ports, allow
+inbound connections on the following:
+
+- 1527 (Derby on your Coordinator; not needed if you are using a separate metadata store like MySQL or PostgreSQL)
+- 2181 (ZooKeeper; not needed if you are using a separate ZooKeeper cluster)
+- 8081 (Coordinator)
+- 8082 (Broker)
+- 8083 (Historical)
+- 8090 (Overlord)
+- 8091, 8100&ndash;8199 (Druid Middle Manager)
```
Comment (xvrl) [inline docs/content/tutorials/cluster.md]: there is technically no upper limit for peon ports.

File: docs/content/tutorials/cluster.md [hunk 2]
```diff
@@ -258,6 +258,25 @@ Keep -XX:MaxDirectMemory >= numThreads*sizeBytes, otherwise Druid will fail to s
...
+## Open ports (if using a firewall)
+
+If you're using a firewall or some other system that only allows traffic on specific ports, allow
+inbound connections on the following:
+
+- 1527 (Derby on your Coordinator; not needed if you are using a separate metadata store like MySQL or PostgreSQL)
+- 2181 (ZooKeeper; not needed if you are using a separate ZooKeeper cluster)
+- 8081 (Coordinator)
+- 8082 (Broker)
+- 8083 (Historical)
```
Comment (xvrl) [inline docs/content/tutorials/cluster.md]: router uses 8888

Comment (xvrl) [inline docs/content/tutorials/cluster.md]: also add realtime nodes on port 8084

**Comment by fjy** [comment]
@xvrl this was already merged

**Comment by fjy** [comment]
also, with new docs, we're de-emphasizing RT nodes a bit

Router is experimental

**Comment by xvrl** [comment]
@fjy just because it has been merged already doesn't mean we can't fix it.

Also, I don't see why we can't be exhaustive on the port list, even if we de-emphasize things elsewhere.
If the worry is about confusing people in the tutorial, then maybe we should put an exhaustive documentation about default ports elsewhere.
I think we can declare router stable as of 0.9 given how long it's been exercised in production.

**Pull request merged**